content/help/faq/security/check-policy.md

```diff
@@ -6,9 +6,11 @@ weight: 11
 The `istioctl` tool provides an option for this purpose. You can do:
 
 {{< text bash >}}
-$ istioctl authn tls-check httpbin.default.svc.cluster.local
+$ istioctl authn tls-check $CLIENT_POD httpbin.default.svc.cluster.local
 HOST:PORT                                  STATUS     SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
-httpbin.default.svc.cluster.local:8000     OK         mTLS       mTLS       default/            default/default
+httpbin.default.svc.cluster.local:8000     OK         mTLS       mTLS       default/            default/istio-system
```

Comment:
Should be `istio-system/default`, not `default/istio-system`

Also needs to be fixed on lines 86 and 90 of mutual-tls/index.md

Reply:
It's `name/namespace`, so `default/istio-system` should be the right one.

Reply:
Also updated mutual-tls/index.md. Thanks for the catch.